getting_started/setup_vm/roles/intel/vars/common.yml

```diff
@@ -20,6 +20,8 @@ intel_sgx_dev_packages:
 intel_dcap_packages:
   - "libsgx-dcap-ql"
   - "libsgx-urts"
+  - "libsgx-quote-ex"
 
 intel_dcap_dev_packages:
   - "libsgx-dcap-ql-dev"
```

Comment:
Deliberately only on dev images, the expectation for the app-run image is that it's run under Kubernetes, where a single externally deployed instance of AESM is expected, eg. https://docs.microsoft.com/en-us/azure/confidential-computing/confidential-nodes-out-of-proc-attestation#how-does-the-out-of-proc-attestation-mode-work-for-confidential-workloads-scenario